ksql-cli/src/test/java/io/confluent/ksql/CliTest.java

```diff
@@ -214,6 +215,21 @@ private static void selectWithLimit(String selectQuery, int limit, TestResult.Or
     test(selectQuery, expectedResults);
   }
 
+  @Test
+  public void testPrint() throws InterruptedException {
+
+    Thread wait = new Thread(() -> {
+        CliTest.this.run("print 'ORDER_TOPIC' FROM BEGINNING INTERVAL 2;", false);
```

Comment:
could you use a `CountdownLatch` here so we don't need the sleep? or does this run forever?